### PR TITLE
doc: follow the Emacs Lisp Style Guide

### PR DESCRIPTION
**Pull request by dalugm**
fix typo: `iff` -> `if`

### CODE REVIEW

**Comment by tomdl89** [comment]
What's wrong with `iff`?

**Comment by dalugm** [comment]
> What's wrong with `iff`?

Redundant, `if` in programming language has the same logic as the `iff` in math.
 
If you want to keep it as is, I can change it back

**Comment by tomdl89** [comment]
In general, I'm not a fan of this sort of PR, especially from a project newcomer. I have to manually check all 123 lines to ensure there's no malicious code, I get drawn into bikeshedding things like `iff` vs `if` and it turns out I still have to double check the changes because some of them make the doc strings grammatically inconsistent. I think my (and your) time would be much better spent on a functional PR. Just my ¢2

**Comment by dalugm** [comment]
> I'm not a fan of this sort of PR, especially from a project newcomer.

Feel free to close this pr if you don't like it, I'm opening this for I found these discrepancies with the style guide when I was reading the evil to configure related packages.

> make the doc strings grammatically inconsistent

Sorry for that, I used a regexp to replace it for simple. I will double check myself in next commit (if you didn't close this).

> ensure there's no malicious code

 As for the malicious code, there is no need to care about, because I only modified the docstring

**Comment by tomdl89** [comment]
> Sorry for that, I used a regexp to replace it for simple. I will double check myself in next commit (if you didn't close this).

No problem - I won't close this. Just wanted to make you aware these sorts of PRs as a newcomer often irritate maintainers. I should have been more positive: I'd love to see PRs from you for functional changes.

> As for the malicious code, there is no need to care about, because I only modified the docstring

That's what someone who was trying to sneak malicious code into a big PR would say 😂 - I need to check that all 123 lines are just docstring modifications. I can't just take your word for it. Again, if you had had a few small functional PRs merged first, I'd be much more trusting. Call me paranoid... 🤷

**Comment by dalugm** [comment]
> had a few small functional PRs merged first, I'd be much more trusting

Ahhh...ok, I'll try this first.

I'll put this on hold and look in the Issues to see if there's anything I can help.

**Comment by tomdl89** [comment]
Ah, I see. Sorry I assume everyone who makes PRs on evil is a programmer. Yes evil is quite complex, but if you're interested enough, you can pick up elisp in a few weeks and start playing around. I think we should be able to merge this PR and your other one soon - I just need some time to review.

File: evil-search.el
Comment (tomdl89) [inline evil-search.el:927]:
```suggestion
Return a triple (regexp offset next-search)."
```

**Review by tomdl89 — APPROVED** [review]
Grammatically, everything now looks good. I'd prefer to keep `iff`s unless the elisp style guide advises against it.

**Comment by dalugm** [comment]
> I'd prefer to keep iffs unless the elisp style guide advises against it.

I'll change it back then.

---

> need some time to review

Just take your time.

> you can pick up elisp in a few weeks and start playing around

Thx, although I use Emacs on my daily life, but most of my
configuration is based on copy&paste and currently my 
knowledge of elisp is very sketchy.

So I guess now should be a good time to drive me learning
elisp seriously.😄

**Review by tomdl89 — APPROVED** [review]

**Pull request merged**
